packages/odyssey-react/src/components/Radio/RadioGroup.module.scss

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
```

Comment:
probably want 2021 here - no big deal really but let me know if you're having trouble with your editor automatically adding these to files for you. I wrote a custom rule to apply the fix automatically via stylelint